deeplake/core/tensor.py

```diff
@@ -714,6 +718,15 @@ def __iter__(self):
                 i, is_iteration=not isinstance(self.index.values[0], list)
             )
 
+    @property
+    def is_empty_tensor(self):
+        if (
+            self.meta.chunk_compression
+            and get_compression_type(self.meta.chunk_compression) != BYTE_COMPRESSION
+        ):
+            return self.meta.max_shape == [0, 0, 0]
```

Comment:
Why do we compare exactly to the list of length 3? Can't max_shape be 2 dimensional array? Let's say 2d masks